### PR TITLE
Fix #8613: get codecov working again

### LINKED ISSUE #8613: Get codecov for tools/ working again
We used to have code coverage reports for tools/. We should get these working again.

### PR DESCRIPTION
**Pull request by gsnedders**
Fixes #8613.

### CODE REVIEW

File: tools/tox.ini [hunk 1]
```diff
@@ -16,12 +16,15 @@ deps =
   pytest-catchlog
 
 commands =
-  pytest --cov {posargs}
+  pytest {posargs:--cov-config={toxinidir}/../.coveragerc --cov}
```
Comment (gsnedders) [inline tools/tox.ini]: Well this is wrong.

**Comment by w3c-bots** [comment]
# Build BROKEN

Started: 2018-02-15 21:59:50
Finished: 2018-02-15 22:17:32


<h2>Failing Jobs</h2>
<ul>

<li>tools_unittest in py27</li>

<li>tools_unittest in pypy</li>

</ul>




View more information about this build on:

- [WPT PR Status](http://pulls.web-platform-tests.org/build/24428)
- [TravisCI](https://travis-ci.org/w3c/web-platform-tests/builds/341998714)

**Review by Ms2ger — APPROVED** [review]
Can't say much about the code, but this seems to work. Do squash.

**Comment by gsnedders** [comment]
https://codecov.io/gh/w3c/web-platform-tests/tree/b3ad93c2c75c553be65ccdc3427a6e4c705c8fcf is the latest built, and various things are broken, I *think* on the codecov.io side.

**Comment by foolip** [comment]
@gsnedders are you still interested in pursuing this?

@LukeZielinski we recently discussed code coverage, here's some prior art.

**Comment by Ms2ger** [comment]
Rebased, let's see what happens.

**Comment by gsnedders** [comment]
We also need to provide an upload token to codecov on TaskCluster, see https://docs.codecov.io/docs/about-the-codecov-bash-uploader.

**Review by Ms2ger — APPROVED** [review]
Happy to put a green tick on this, but I'm not able to review correctness.

**Comment by jgraham** [comment]
I think we're not doing this now.

**Pull request closed without merging**